mkdocs/themes/mkdocs/main.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{#
+The entry point for the MkDocs Theme.
+ 
+Any custom themes which inherit from this theme should override this file to redefine
```

Comment:
How about changing to "Any theme customisations should override this file..."

There are two subtly different use cases. External custom themes and theme customisations, I want to avoid the two being confused.